config/manager/kustomization.yaml

```diff
@@ -1,3 +1,9 @@
 resources:
 - enoConfigMap.yaml
 - enoDeployment.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: eno-controller
+  newTag: latest
```

Comment:
This is automatically generated when you run the make deploy command. I think we should remove it from here as it is automatically generated.

Reply:
I will run this in my cluter and check if this is working properly.

Reply:
AS per the discussion we have decided to push this file.